ntbs-service/Pages/Index.cshtml

```diff
@@ -5,86 +5,158 @@
     ViewData["Title"] = "Home page";
     Layout = "./Shared/_NoForm_Layout.cshtml";
 }
+<tb-service-filtered-alerts inline-template>
+    <div>
+        <div class="flex-container">
+            <h2 class="alerts-heading"> Alerts </h2>
+            <label nhs-label-type="Standard" for="tb-service-filter-dropdown" classes="tb-service-alerts-filter-label">
+                Filter by TB Service
+            </label>
+            <select id="tb-service-filter-dropdown" nhs-select-type="Standard" ref="tbService" asp-items="Model.TbServices" v-on:change="updateTable" classes="tb-service-alerts-filter-dropdown">
+                <option> Show all </option>
+            </select>
```

Comment:
Let's make sure these only appear when they are functional, i.e. when js is available